Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.cpp

```diff
@@ -69,6 +70,24 @@ namespace WhiteBox
         AzFramework::EntityDebugDisplayEventBus::Handler::BusDisconnect();
     }
 
+    void EditorWhiteBoxComponentMode::UpdateTransformCluster()
+    {
+        bool enable = false;
+        switch (m_currentSubMode)
+        {
+        case SubMode::Default:
+        case SubMode::EdgeRestore:
+            enable = false;
+            break;
+        case SubMode::Transform:
+            enable = true;
+            break;
+        }
+        AzToolsFramework::ViewportUi::ViewportUiRequestBus::Event(
+            AzToolsFramework::ViewportUi::DefaultViewportId, &AzToolsFramework::ViewportUi::ViewportUiRequestBus::Events::SetClusterVisible,
+            m_transformClusterId, enable);
```

Comment:
suggestion: In future we might want to actually add a new button for `SubMode::Transform` instead of hiding them, I'd recommend we move to this if we can

Reply:
(sorry kept reading and I think I see what is happening sorry 😅 this actually could be okay (I'll check out the video soon too)